lib/tasks/seed_pokemons.rake

```diff
@@ -0,0 +1,32 @@
+require 'httparty'
```

Comment:
Tudo bem fazer com rake, mas a idéia inicial era colocar no `db/seeds.rb`

Assim, quando der o comando rails db:seed, ele popularia a base de dados


db/seeds.rb

```
# This file should ensure the existence of records required to run the application in every environment (production,
# development, test). The code here should be idempotent so that it can be executed at any point in every environment.
# The data can then be loaded with the bin/rails db:seed command (or created alongside the database with db:setup).
#
# Example:
#
#   ["Action", "Comedy", "Drama", "Horror"].each do |genre_name|
#     MovieGenre.find_or_create_by!(name: genre_name)
#   end

pokemon_api_url = "https://pokeapi.co/api/v2/pokemon"
species_api_url = "https://pokeapi.co/api/v2/pokemon-species"

response = HTTParty.get("#{pokemon_api_url}?limit=200")
pokemons = response.parsed_response['results']

pokemons.each do |pokemon|
  pokemon_data = HTTParty.get(pokemon['url']).parsed_response
  species_data = HTTParty.get("#{species_api_url}/#{pokemon_data['id']}").parsed_response

  name = pokemon_data['name']
  types = pokemon_data['types'].map { |t| t['type']['name'] }
  image = pokemon_data['sprites']['front_default']
  moves = pokemon_data['moves'].map { |m| m['move']['name'] }
  description = species_data['flavor_text_entries'].find { |entry| entry['language']['name'] == 'en' }['flavor_text']

  Pokemon.create!(
    nome: name,
    tipo: types,
    imagem: image,
    moves: moves
  )
end

puts "Seeded #{pokemons.count} Pokémon"
```